lib/tasks/bootdisk.rake

```diff
@@ -45,6 +45,18 @@ namespace :bootdisk do
         puts "Wrote #{output}"
       end
     end
+
+    desc 'Generate a subnet disk for a specific subnet. NAME=subnet, OUTPUT=path'
+    task :subnet => :environment do
+      subnet = Subnet.find_by_name(ENV['NAME']) || raise("cannot find subnet '#{ENV['NAME']}', specify NAME=subnet")
```

Comment:
This rake task follows shell-like syntax, its in the whole file so I just align with this style.

The other ones fixed, pushed.